packages/hop-node/src/watchers/SettleBondedWithdrawalWatcher.ts

```diff
@@ -131,7 +131,7 @@ class SettleBondedWithdrawalWatcher extends BaseWatcher {
 
         // check settle-able transfer root
         promises.push(
-          this.checkTransferRootHash(transferRootHash!, bonder) // eslint-disable-line @typescript-eslint/no-non-null-assertion
+          this.checkTransferRootId(transferRootId!, bonder) // eslint-disable-line @typescript-eslint/no-non-null-assertion
             .catch((err: Error) => {
               this.logger.error('checkTransferRootHash error:', err.message)
```

Comment:
This should say `checkTransferRootId error`

Reply:
Updated in https://github.com/hop-protocol/hop/pull/318/commits/48bcdaf9078daa59da9e74ca6af291aec5d1489c